fiorella/src/sections/body-message.liquid

```diff
@@ -0,0 +1,36 @@
+<div class="body-message">
+  <h1 class="title-message">{{ section.settings.titlemessage | upcase }}</h1>
+  <img class="image-message" src="{{ section.settings.imagemessage | img_url: '800x' }}" alt="{{ settings.categoryimg.alt }}" />
+  <div class="text-message">
+    <p>{{ section.settings.imagedescription }}</p>
+  </div>
+  <a href="{{section.settings.link}}" class="btn-read-more">READ MORE</a>
```

Comment:
Can you change this text with text from locale file?

Why use locale files?
When your app supports internationalize language you need locales files, this way you app pull all text from one single file.

Example:
<p>{{ 'homepage.footer.follow_us' | t }}